code/game/turfs/simulated/chasm.dm

```diff
@@ -22,6 +22,9 @@
 
 /turf/open/chasm/proc/drop_stuff(AM)
 	. = 0
+	for(var/obj/O in contents)
```

Comment:
:-1: :x: :100: implement a safeties system like I did for lava (copy it if you want, it's 1 proc and like 4 other lines)